README.md

```diff
@@ -128,6 +128,10 @@ If we want our earnings to be in `tokenA` then the
 commodity must be `tokenB`. So we can buy from the sell order, `20 tokenB` using `8 tokenA`, then
 using these `20 tokenB` we can get `10 tokenA` from the buy order, earning `2 tokenA`.
 
+> **ⓘ **
+>
+> Note that as per current design of bot, there is a check in end to determine whether the bot would loose balance in case it submits the build transaction. This check subtracts for transaction fees, that is, it might happen that bot may loose ADA when submitting the transaction but that would solely be due to transaction fees. If currency is set to ADA, bot may still loose some ADA if the transaction fees is more than found ADA arbitrage.
```

Comment:
> the bot would loose balance

I guess you meant to write "the bot would **lose** balance"

Reply:
Yes, thanks for pointing it out @adacapo21!